tests/test_client.py

```diff
@@ -1,9 +1,15 @@
 import os
 import sys
+import base64
+import re
 import inspect
+import hashlib
+from collections import OrderedDict
+from six.moves import socketserver
```

Comment:
`six` is a dependency of pytest anyway. It won't be installed with the library, but it will be present when the tests are run.